### PR TITLE
Executive summary

### PR DESCRIPTION
**Pull request by ronaldo1618**
adding summary

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -48,12 +49,56 @@ a {
...
-  flex-direction: row;
+  position: 0;
+  bottom: 0;
+  width: 100%;
+  flex-direction: row-reverse;
 }
 
 .snicons a {
   object-fit: contain;
+}
```
Comment (CabraArretado) [inline styles/main.css:63]: I think we should create different CSS files with the classes we use just in our pages

**Review by CabraArretado — COMMENTED** [review]
I think we should create different CSS files with the classes we use just in our pages

**Comment by CabraArretado** [comment]
👍

**Comment by danny128373** [comment]
👍

**Pull request merged**